src/api/client.js

```diff
@@ -18,10 +25,31 @@ function assetTypesFromState(assetTypes) {
   return Object.keys(assetTypes).filter(key => assetTypes[key]);
 }
 
-export function requestAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetTypes = {} }) {
-  const assetTypesToFilter = assetTypesFromState(assetTypes);
+function getDatabaseAttributesFromAssetAttributes(sort) {
+  return assetAttributesToDatabaseAttributes[sort];
+}
+
+function objectIsNonEmpty(object) {
+  return Object.keys(object).length > 0;
+}
+
+export function requestAssets(courseId, params) {
+  const assetTypesToFilter = assetTypesFromState(params.assetTypes);
+  const sortType = getDatabaseAttributesFromAssetAttributes(params.sort);
```

Comment:
This converts the sortType from the attribute names returned by the API to the attribute names expected by the API.